studies/RenderDocument.json5

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    name: 'RenderDocument',
+    experiment: [
+      {
+        name: 'Level_Subframe',
+        probability_weight: 100,
+        feature_association: {
+          enable_feature: [
+            'RenderDocument',
+          ],
+        },
+        param: [
+          {
+            name: 'level',
+            value: 'subframe',
+          },
+        ],
+      },
+    ],
+    filter: {
+      min_version: '142.1.84.141',
+      channel: [
+        'RELEASE',
+        'NIGHTLY',
+        'BETA',
+      ],
+      platform: [
+        'WINDOWS',
+        'MAC',
```

Comment:
While it's not enabled on macOS or Android right now, we should prevent a scenario where upstream does it via Finch and we're caught unawares.